distributor-widget/advanced-guide.md

```diff
@@ -97,15 +97,9 @@ The deeplinks are also supported by standalone Distributor.
 
 Payment gateway is used to safely collect information about a customer’s credit card. A configuration is done once, when the hotel is set up. Distributor would use it automatically. Currently Distributor supports these gateways:
 
-* [Braintree](https://www.braintreepayments.com/)
```

Comment:
Was this outdated?

Reply:
as @mveith mentioned PCI Proxy is no longer a gateway but `payment card storage` so we might want to fix the title and description of this section as well